mbedtls-sys/build/bindgen.rs

```diff
@@ -37,37 +86,31 @@ impl super::BuildConfig {
 
         let include = self.mbedtls_src.join("include");
 
-        let logger = StderrLogger;
-        let mut bindgen = bindgen::Builder::new(header.into_os_string().into_string().unwrap());
-        let bindings = bindgen
-            .log(&logger)
+        let bindings = bindgen::builder()
             .clang_arg("-Dmbedtls_t_udbl=mbedtls_t_udbl;") // bindgen can't handle unused uint128
             .clang_arg(format!(
                 "-DMBEDTLS_CONFIG_FILE=<{}>",
                 self.config_h.to_str().expect("config.h UTF-8 error")
             )).clang_arg(format!(
                 "-I{}",
                 include.to_str().expect("include/ UTF-8 error")
-            )).match_pat(include.to_str().expect("include/ UTF-8 error"))
-            .match_pat(self.config_h.to_str().expect("config.h UTF-8 error"))
-            .use_core(true)
+            )).header(
+                header
+                    .to_str()
+                    .expect("failed to convert header path to string"),
+            ).use_core()
             .derive_debug(false) // buggy :(
-            .ctypes_prefix(vec!["types".to_owned(), "raw_types".to_owned()])
-            .remove_prefix("mbedtls_")
-            .rust_enums(false)
-            .convert_macros(true)
-            .macro_int_types(
-                vec![
-                    "sint",
-                    "sint",
-                    "sint",
-                    "slonglong",
-                    "sint",
-                    "sint",
-                    "sint",
-                    "slonglong",
-                ].into_iter(),
-            ).generate()
+            .disable_name_namespacing()
+            .prepend_enum_name(false)
+            .ctypes_prefix("raw_types")
+            .parse_callbacks(Box::new(ParseCallbacks{}))
+            // max_align_t is causing bindgen generated tests to fail an alignment check and
+            // is not needed by the bindings.
+            .blacklist_type("max_align_t")
+            // Including the comments breaks the generated code because it contains formatting
+            // that is interpreted as escaped characters.
```

Comment:
The mbedtls api comments contain tokens like '\brief', '\param' etc. The generated bindings.rs fail to compile and the errors show that the compiler is trying to interpret the comments are rust code. I will try to pinpoint what token is triggering that, it's a bit difficult because rustc is generated many pages of errors.

Reply:
Ok, that's fine. Don't worry about it.